static/js/settings_bots.js

```diff
@@ -28,7 +28,10 @@ function render_bots() {
     $('#active_bots_list').empty();
     $('#inactive_bots_list').empty();
 
-    _.each(bot_data.get_all_bots_for_current_user(), function (elem) {
+    var all_bots_for_current_user = bot_data.get_all_bots_for_current_user();
+    var is_all_inactive_bots = true;
```

Comment:
I'd negate the meaning of this, to "user_owns_an_active_bot" defaulting to false; then you don't need to do the length check later, so it'll be cleaner.